static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
I'm happy to do it—yes, I agree we should question all the choices ever made on code that is not "ours", and critique is second to none when it comes to improving things—but I find no compelling case for it. I may be missing the point. Why don't we inline styles for all similar small components then?